### PR TITLE
update to jammy

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -11,7 +11,7 @@ envlist = fmt, lint, unit
 src_path = {toxinidir}/src/
 tst_path = {toxinidir}/tests/
 lib_path = {toxinidir}/lib/charms/mongodb
-all_path = {[vars]src_path} {[vars]tst_path}
+all_path = {[vars]src_path} {[vars]tst_path} {[vars]lib_path}
```
Comment (taurus-forever) [inline tox.ini:14]: Why do we play this games? The following pass lint without any issues:

```suggestion
all_path = {toxinidir}
```

**Review by taurus-forever — APPROVED** [review]
Nice! One minor suggestion.

File: tox.ini (hunk 1 above)
Comment (delgod) [reply tox.ini:14]: we don't want to run checks against fetched 3rd-party libraries

File: tests/integration/tls_tests/test_tls.py [hunk 2]
```diff
@@ -27,11 +27,16 @@ async def test_build_and_deploy(ops_test: OpsTest) -> None:
...
+        await ops_test.model.deploy(my_charm, num_units=3, resources=resources, series="jammy")
         await ops_test.model.wait_for_idle(apps=[DATABASE_APP_NAME], status="active", timeout=2000)
 
         config = {"generate-self-signed-certificates": "true", "ca-common-name": "Test CA"}
-        await ops_test.model.deploy(TLS_CERTIFICATES_APP_NAME, channel="beta", config=config)
+        await ops_test.model.deploy(
+            TLS_CERTIFICATES_APP_NAME,
+            channel="beta",
+            config=config,
+            series="focal",
```
Comment (MiaAltieri) [inline tests/integration/tls_tests/test_tls.py:38]: I assume TLS-certs is not on jammy yet?

Comment (MiaAltieri) [inline tests/integration/tls_tests/test_tls.py:38]: I assume that TLS-certs isn't availbe on jammy yet

**Review by MiaAltieri — APPROVED** [review]
groundbreaking 🍯 🐟 (supposed to be jelly-fish)

File: tests/integration/tls_tests/test_tls.py (hunk 2 above)
Comment (delgod) [reply tests/integration/tls_tests/test_tls.py:38]: you right, tls-certificates-operator is on focal for now

**Pull request merged**